### DISCUSSION TITLE
Restricting file types for field interface

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#15404] [Archive]
Is there a way to restrict file types for file/multifile interfaces? There's field validation but the file must have already been uploaded by then. Considering creating an extension as a last resort

**Comment by azrikahar** [comment]
There's currently no way to do so, but there are several discussion regarding this topic. I believe #11733 would be the highest voted among the rest, so feel free to give it an upvote. 

In the meantime, a custom extension would be the interim solution indeed, as described by this comment: https://github.com/directus/directus/discussions/11733#discussioncomment-2218624

(also technically currently field conditions & validation does not 100% work for relational fields, but we're tracking them over in #8228 and #14414 respectively)

---

Linking related discussions for future reference: #14457 #12790 #6097
